presto-main/src/main/java/com/facebook/presto/server/security/LdapServerConfig.java

```diff
@@ -69,4 +78,40 @@ public LdapServerConfig setAuthenticationEnabled(boolean enabled)
         this.authenticationEnabled = enabled;
         return this;
     }
+
+    public String getGroupDistinguishedName()
```

Comment:
This config should only be available for Active Directory. You shouldn't be able to configure it if it won't be used.